spec/services/customers/update_service_spec.rb

```diff
@@ -33,6 +34,7 @@
       updated_customer = result.customer
       aggregate_failures do
         expect(updated_customer.name).to eq('Updated customer name')
+        # expect(updated_customer.timezone).to eq('Europe/Paris') # TODO: Timezone update is turned of for now
```

Comment:
What about having a naming like `TODO(:timezone)` so that it will be easier for searching?